tests/dotnet/UnitTests/ProjectTest.cs

```diff
@@ -747,5 +747,29 @@ public void KillEverything ()
 			// https://github.com/xamarin/xamarin-macios/issues/10012
 			ExecutionHelper.Execute ("xcrun", new [] { "simctl", "list" });
 		}
+
+
+		[TestCase (ApplePlatform.iOS, "MyShareExtension.csproj", "bin/Debug/net6.0-ios/iossimulator-x64/MySimpleApp.app/PlugIns/MyShareExtension.appex")]
+		[TestCase (ApplePlatform.TVOS, "MyTVServicesExtension.csproj", "bin/Debug/net6.0-tvos/tvossimulator-x64/MySimpleApp.app/PlugIns/MyTVServicesExtension.appex")]		
+		[TestCase (ApplePlatform.MacOSX, "ShareExtensionTest.csproj", "bin/Debug/net6.0-macos/osx-x64/MySimpleApp.app/Contents/PlugIns/ShareExtensionTest.appex")]
```

Comment:
As mentioned before, if you name all the extension projects the same way, you can the platform-specific special casing here.